src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -233,6 +234,11 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         public bool RefreshOnIssuerKeyNotFound { get; set; }
 
         /// <summary>
+        /// Determines whether authorization requests should be made using POST.
+        /// </summary>
+        public bool UsePostAuthorizationRequests { get; set; }
```

Comment:
Lets not go there yet. It wouldn't even be an OIDC specific option, it would be an OIDC challenge specific option.  RedirectToIdentityProvider is also used for SignOut.

That said, does the protocol support POSTs for SignOut? It does not appear to: http://openid.net/specs/openid-connect-session-1_0.html#RPLogout